### ISSUE TITLE
sensuctl entity examples with namespace 'web' should be consistent

### ISSUE THREAD

**Issue by candlerb** [sensu/sensu-docs#1214]
## This is an issue with:

- [ ] Bug (site functionality or styling)
- [X] Errata (fix needed for doc content)
- [ ] New content (guide wanted)
- [ ] Update (Add missing or refresh existing content)
- [ ] Enhancement (add new site functionality)

## Expected Behavior

At https://docs.sensu.io/sensu-go/5.2/reference/entities/#managing-entity-labels it shows creating a proxy entity in namespace "web".

```
{
  "type": "Entity",
  "api_version": "core/v2",
  "metadata": {
    "name": "sensu-docs",
    "namespace": "web",
    "labels": {
      "url": "docs.sensu.io"
    }
    ...
```

The examples should work if reproduced by the user, but currently give errors.

## Current Behavior

1. Entity can't be created unless you've first done `sensuctl namespace create web` (not a big deal)
2. The part where it explains editing labels shows `sensuctl edit entity sensu-docs`, but I think it should be `sensuctl edit entity sensu-docs --namespace web`
3. Even that doesn't work - it gives "Error: not found"

```
[sensu_go_sandbox]$ sensuctl edit entity sensu-docs --namespace web
Error: not found
Error: couldn't get resource: exit status 1
[sensu_go_sandbox]$ sensuctl edit entity sensu-docs
Error: not found
Error: couldn't get resource: exit status 1
[sensu_go_sandbox]$ sensuctl entity list --namespace web
      ID       Class   OS   Subscriptions   Last Seen
 ──────────── ─────── ──── ─────────────── ───────────
  sensu-docs   proxy                        N/A
[sensu_go_sandbox]$ sensuctl entity info sensu-docs --namespace web --format json
{
  "entity_class": "proxy",
  "system": {
    "network": {
      "interfaces": null
    }
  },
  "subscriptions": [],
  "last_seen": 0,
  "deregister": false,
  "deregistration": {},
  "metadata": {
    "name": "sensu-docs",
    "namespace": "web",
    "labels": {
      "url": "docs.sensu.io"
    }
  }
}
```

I'm not sure if this is an issue with the docs (maybe the fact that the entity hasn't been seen means you can't edit it?) or a code bug (`sensu edit entity` with namespace doesn't work?)

It's certainly odd that `sensuctl entity info` works but `sensuctl edit entity` does not.

## Possible Solution

Add `--namespace web` to the examples where required.  Add explanation for why `sensuctl edit entity` gives an error. 

## Steps to Reproduce (for bugs)

See above

## Screenshots (if appropriate):

See above

## Context

* Browser(s) and version(s): N/A (reproduce from CLI)
* OS: macOS 10.14.3 with vagrant sensu_go_sandbox

**Comment by apaskulin** [comment]
Thanks for opening this issue, @candlerb! I'm seeing the same errors. In order to run `sensuctl edit entity sensu-docs`, I needed to first re-run `sensuctl configure` and apply the `web` namespace.

For the sake of clarity, I think we should change the docs to use the default namespace, but I agree that I would expect `sensuctl edit entity sensu-docs --namespace web` to work. Let me know if you'd like me to open an issue in the Sensu Go project to track this issue.

**Comment by candlerb** [comment]
Yes, I think it would be good to open an issue for this, especially since the built-in help for `sensuctl edit` says that `--namespace` is a valid option (and it's accepted).

As for changing the examples to use default namespace, I'm fine with that too.  I thought maybe the intention was to give an example of namespaces in action, but there's no particular need to do that here.

**Comment by rgeniesse** [comment]
I've opened: https://github.com/sensu/sensu-go/issues/2739 to deal with the flag behavior described here.

I did some further testing and found that the flag doesn't appear to be honored at all, taking a `default` value no matter what. I put the entity into the `default` namespace and used `--namespace web` and it found the entity.

That said, I've got a PR open to also move the entity reference example back into the default namespace. In this case, a separate one isn't required to get the concept the documentation is describing across.
